scripts/upload_test_contract.sh

```diff
@@ -33,3 +33,9 @@ echo $KEYRING_PASS | desmos --keyring-backend=file tx wasm instantiate 1 "{}" \
   --from $USER1 --label "test-contract" --admin $USER1_ADDRESS --chain-id=testchain -b=sync -y
 echo "Contract initialized"
 
+# Initialize the second contract
+echo "Initializing second contract..."
+echo $KEYRING_PASS | desmos --keyring-backend=file tx wasm instantiate 1 "{}" \
+  --from $USER1 --label "test-contract-" --admin $USER1_ADDRESS --chain-id=testchain -b=sync -y
+echo "Second contract initialized"
```

Comment:
It is for testing receiver of `MsgAcceptPostOwnerTransferRequest` and `MsgRefusePostOwnerTransferRequest` contract, the second contract is used as receiver.